waterloop-site/src/components/SustainableTech/NavBar/LinkScroll.tsx

```diff
@@ -0,0 +1,11 @@
+// Moves user to the desired section of the webpage
+function scrollDown(id : string){
```

Comment:
Can you change this to `id: string`? There are a few formatting issues throughout the different files you've worked on. Could you make sure to run your files through `format document` with prettier?